src/service.py

```diff
@@ -88,7 +101,7 @@ def workaround_bug_268() -> None:
     https://github.com/openstack-exporter/openstack-exporter/issues/268
     """
     logger.info("Adding service override to workaround bug 268")
-    dir = "/etc/systemd/system/snap.golang-openstack-exporter.service.service.d"
+    dir = "/etc/systemd/system/snap.charmed-openstack-exporter.service.service.d"
     os.makedirs(dir, exist_ok=True)
     with open(f"{dir}/bug_268.conf", "w") as f:
         f.write("[Service]\nEnvironment=OS_COMPUTE_API_VERSION=2.87\n")
```

Comment:
**nitpick (blocking):** 

Suggest to move these line to a independent function.

Reply:
They are in an independent function already - it's in `workaround_bug_268()`.